services/net/contentmigration/migrators/ClipMigrator.cs

```diff
@@ -64,18 +65,18 @@ public ClipMigrator(IApiService api, IOptionsSnapshot<MigratorOptions> migratorO
             // FilePath = (ingest.DestinationConnection?.GetConfigurationValue("path")?.MakeRelativePath() ?? "")
             //     .CombineWith($"{ingest.Source?.Code}/{GetDateTimeForTimeZone(ingest):yyyy-MM-dd}/", reference.Uid),
             Language = "", // TODO: Need to extract this from the ingest, or determine it after transcription.
-            //Tags = item.Categories.Select(c => new TNO.Kafka.Models.Tag(c.Name, c.Label))
-
         };
 
+        var auditUser = lookups.Users.FirstOrDefault(u => u.Username == this.Options.DefaultUserNameForAudit);
```

Comment:
This could return null.  Throw a configuration exception to inform us something isn't configured correctly.

Reply:
definitely will do.